holesky/run.sh

```diff
@@ -45,7 +102,6 @@ listQuorums() {
   # we have to pass a dummy quorum-id-list as it is required by the plugin
   docker run --env-file .env \
     --rm \
-    --volume "${NODE_ECDSA_KEY_FILE_HOST}":/app/operator_keys/ecdsa_key.json \
```

Comment:
If this cmd is run from CLI machine already, this flag seems fine. Or maybe we could also remove the `NODE_ECDSA_KEY_PASSWORD` below?

Reply:
ah yea I would remove password. I dont' want to them to pass any ecdsa key details from list quorums